packages/graphql-lowcode/src/modify/addFragmentFields.ts

```diff
@@ -0,0 +1,43 @@
+import { parse, visit, print } from 'graphql'
+
+export function addFragmentField(graphqlQuery: string, options: { entity: string, property: string }) {
+  if (!graphqlQuery.indexOf('fragment')) return graphqlQuery
+  const format = graphqlQuery.indexOf('\n') >= 0 ? '\n' : ''
+
+  const ast = parse(graphqlQuery)
+  let finalQuery = graphqlQuery
+
+  let lastFragmentFieldIndexes: number[] = []
+
+  //finds the first occurence of '{' and counts spaces after, to set number of indent spaces
+  const firstBracket = graphqlQuery.indexOf('{') + 1
+
+  const indentSize = format === '\n' ? 2 : 1
+  let indent = 0
+
+  for (const char of graphqlQuery.substr(firstBracket)) {
```

Comment:
Please move the code that guess indentation in a specific function. This will help another coder to read the code more intuitively.